src/test/java/net/snowflake/ingest/streaming/internal/FlushServiceTest.java

```diff
@@ -35,10 +34,7 @@
 import net.snowflake.client.jdbc.SnowflakeConnectionV1;
 import net.snowflake.client.jdbc.internal.org.bouncycastle.jce.provider.BouncyCastleProvider;
 import net.snowflake.ingest.streaming.OpenChannelRequest;
-import net.snowflake.ingest.utils.Cryptor;
-import net.snowflake.ingest.utils.ErrorCode;
-import net.snowflake.ingest.utils.ParameterProvider;
-import net.snowflake.ingest.utils.SFException;
+import net.snowflake.ingest.utils.*;
```

Comment:
Please expand the imports instead of using * imports (according to SF's best coding practice)